src/Main.java

```diff
@@ -6,7 +6,7 @@ public static void main(String[] args) {
         String [] ipSplitters = ip.split("\\.");
         for (String i:ipSplitters){
             System.out.println(i);
-        }
+        } else ( System.out.println (ERROR))
```

Comment:
Programmatically wrong is at considerd else statement with a previous if statement 😉
but great gob 😊